platform/CMakeLists.txt

```diff
@@ -10,4 +10,4 @@ find_library(LIB_RT rt REQUIRED)
 add_subdirectory( ${PLATFORM_DIR}/posix )
 
 # Add ota porting targets
-# add_subdirectory( ${PLATFORM_DIR}/posix/ota_pal )
+add_subdirectory( ${PLATFORM_DIR}/posix/ota_pal )
```

Comment:
If this is causing the unit test coverage to fail, should it be left commented out?

Reply:
Yes, This is being addressed currently and will be introduced in a new PR. For now we would need to link the ota_pal to enable building of the demos.

Reply:
The unit test check is required for merging, a repo admin would have to disable it